Sources/BrowserServicesKit/PrivacyConfig/PrivacyConfigurationData.swift

```diff
@@ -171,6 +187,13 @@ public struct PrivacyConfigurationData {
                 } else {
                     self.rollout = nil
                 }
+
+                if let cohortData = json[CodingKeys.cohorts.rawValue] as? [[String: Any]] {
+                    let parsedCohorts = cohortData.compactMap { Cohort(json: $0) }
+                    self.cohorts = parsedCohorts.isEmpty ? nil : parsedCohorts
+                } else {
+                    self.cohorts = nil
```

Comment:
selfs might be not needed here

Reply:
it's different because self is needed there (because of if let rollout), here it's not, I'd get rid of it